field/src/types.rs

```diff
@@ -563,6 +567,11 @@ pub trait PrimeField64: PrimeField + Field64 {
 }
 
 /// An iterator over the powers of a certain base element `b`: `b^0, b^1, b^2, ...`.
+///
+/// This iterator panics on calls to [`nth`] if `n` overflows [`u64`].
```

Comment:
I'm not sure I understand this comment. The argument of `nth` is `usize`, so how can it overflow `u64`?

Reply:
You're right that this is probably an unnecessary comment. If we decide to keep `nth` I'll get rid of the comment. If you want to get really into the pointer-sized weeds, [this](https://faultlore.com/blah/fix-rust-pointers/#distinguish-pointers-and-addresses) is a great read.